app/controllers/contact_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
         GroupMailer.anonymous_mail(mail_array, contact_params[:body], contact_params[:title]).deliver_now
       end
 
-     redirect_to :contact_index
+      redirect_to :contact_index, flash: {notice: "your mail has been sent to " + contact_params[:to_whom].first}
```

Comment:
Yeah sure, but you should (I believe) also be able to email several committees, all of those should show up, don't you think?